docs/database/updates/2016-12-22_1-schema.sql

```diff
@@ -0,0 +1,24 @@
+CREATE TABLE `audios` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `sentence_id` int(11) NOT NULL,
+  `user_id` int(11) DEFAULT NULL,
+  `external` varchar(500) DEFAULT NULL,
+  `created` datetime NOT NULL,
+  `modified` datetime NOT NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
```

Comment:
We need an index on the column `sentence_id`.

I started deploying this PR on the dev website yesterday and launched the Sphinx indexation. It was still not done today. After some investigation I figured out that the new joins for audio in the sphinx query made it really slow. The query to retrieve and index the English sentences was running after 10 hours and still running.

I did a `ALTER TABLE audios ADD KEY `sentence_id` (`sentence_id`);` and now it's indexing at a normal speed.